test/functionalTest/cases/1289_exceed_mongo_key_limit/exceed_mongo_key_limit.test

```diff
@@ -0,0 +1,86 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I think is better to use the existing folder (1289_exceed_mongo_key_limit/) and create a new .test file inside it (exceed_mongo_key_limit_v1.test) as it is part of the same topic/issue and using two folders will lead to an excesive "fragmentation".